tests/armature/Model.spec.ts

```diff
@@ -15,7 +15,15 @@ describe('Model', () => {
                     .point('base')
                     .attach(Shape.sphere())
             );
-            const exported = model.exportOBJ('model', RGBColor.fromHex('#000000'));
+
+            let exported: Export | undefined;
+            const iter = model.exportOBJIterator('model', RGBColor.fromHex('#000000'));
+            for (;;) {
+                exported = iter.next().value;
```

Comment:
I think our test suite also supports async tests by adding a `done` param, so if we want to keep the iterator generator private, we could do something like:

```typescript
it('does a thing', (done) => {
  // ...
  model.exportObj().then((obj) => {
    expect(something);
    done();
  })
});
```

Reply:
Feel free to ignore if we want to test that the iterator itself works though!

Reply:
The main issue I had was that the task that is started calls `window.requestAnimationFrame` which I don't think makes sense in the context of a headless test.

Reply:
ahhh right good point.